src/main/java/ro/fortsoft/pf4j/update/UpdateManager.java

```diff
@@ -203,31 +203,6 @@ public synchronized void refresh() {
     }
 
     /**
-     * Installs a plugin given only its URL
-     * @param url the url of the plugin to install
-     * @return true if plugin was installed
-     * @deprecated - use the installPlugin(id, version) instead
-     */
-    public boolean installPlugin(String url) {
```

Comment:
It's OK to remove it. In this situation is it OK the signature of `updatePlugin` with `URL url`?